source/options.html

```diff
@@ -82,6 +82,14 @@
 
 	</details>
 
+	<details id="export">
+		<summary><strong>🗄️ Export options</strong></summary>
+		<p>
+			<button type="button" class="js-export">Export</button>
```

Comment:
I am assuming these are per domain.

If yes it should say somewhere that the export/import is for that domain

Reply:
Good idea. There's currently no way to display the current domain other than in the domain selector, so I added some copy

<img width="543" alt="Screenshot 9" src="https://user-images.githubusercontent.com/1402241/230777215-0e40b2d7-0eff-46c7-a530-6399b84cb7f6.png">

Reply:
I was going to ask you of we should add  an option to exclude the token.

But what you did is perfect 👌

Reply:
Thought so too but there's enough spaghetti on this plate 😅 

Since it's just a JSON they can alter it afterwards too. 

Shipping!